src/main/webapp/view/index.html

```diff
@@ -11,12 +11,12 @@
   integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
   crossorigin="anonymous"
 />
-<script type="module" src="/synchronizer.js" defer></script>
-<script type="module" src="/timestamps.js" defer></script>
-<script type="module" src="/view/discussion/discussion.js" defer></script>
-<script type="module" src="/view/video/video.js" defer></script>
-<script type="module" src="/view/transcript/transcript.js" defer></script>
-<script type="module" src="/view/view.js" defer></script>
+<script type="module" src="/Synchronizer.js" defer></script>
+<script type="module" src="/Timestamps.js" defer></script>
+<script type="module" src="/view/discussion/Discussion.js" defer></script>
+<script type="module" src="/view/video/Video.js" defer></script>
+<script type="module" src="/view/transcript/Transcript.js" defer></script>
+<script type="module" src="/view/View.js" defer></script>
```

Comment:
I think you're missing the script tag for `Lectures.js`. It should be in a different index.html